android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -44,17 +44,6 @@ public void prepare(final String fileName, final Integer key, final Callback cal
       callback.invoke(e);
       return;
     }
-    try {
-      player.prepare();
```

Comment:
@benvium I wonder why remove this lines. Maybe for this i can't play audio on android devices.